datahub-web-react/src/app/ingest/source/builder/RecipeForm/csv.ts

```diff
@@ -0,0 +1,43 @@
+import { RecipeField, FieldType } from './common';
+
+export const CSV_FILENAME: RecipeField = {
+    name: 'filename',
+    label: 'File name',
+    tooltip: 'File path or URL of CSV file to ingest.',
+    type: FieldType.TEXT,
+    fieldPath: 'source.config.filename',
+    placeholder: 'File name',
+    required: true,
```

Comment:
let's specify here that it is just a URL. from UI-ingestion end users cannot provide filenames. We should only support urls.